app/src/main/res/layout/include_system_search_onboarding.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<!--
+  ~ Copyright (c) 2018 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/onboarding"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?attr/colorPrimary"
+    android:clipToPadding="false"
+    android:elevation="4dp"
```

Comment:
This elevation seems like it should be the same one as in `activity_system_search-> resultsContent`. If so, we could move that value into a `dimen` to ensure consistency.

Edit: consider using `systemSearchResultsElevation`

Reply:
They happen to be the same by coincidence, we played with many values e.g 2dp for one of them and 4dp for the other and landed on them both being 4dp but I wouldn't want to tightly couple them by using the same variable. They are different components and there is no reason for them to have the same elevation.

Reply:
Agree, if they are not related, we should avoid using the same reference 👍